merkle/sparse_merkle_tree_test.go

```diff
@@ -138,14 +139,16 @@ func getRandomNonRootNode(t *testing.T, rev int64) storage.Node {
 }
 
 func TestRootAtRevision(t *testing.T) {
+	ctx := context.Background()
+
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
 	r, tx := getSparseMerkleTreeReaderWithMockTX(mockCtrl, 100)
 	node := getRandomRootNode(t, 14)
 	tx.EXPECT().Commit().AnyTimes().Return(nil)
-	tx.EXPECT().GetMerkleNodes(int64(23), rootNodeMatcher{}).Return([]storage.Node{node}, nil)
-	root, err := r.RootAtRevision(23)
+	tx.EXPECT().GetMerkleNodes(gomock.Any(), int64(23), rootNodeMatcher{}).Return([]storage.Node{node}, nil)
```

Comment:
In these tests can we check that the supplied context was propagated rather than use Any? Or does that not work.